src/Microsoft.Sbom.Api/Manifest/ManifestConfigHandlers/SPDX30ManifestConfigHandler.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Sbom.Common;
+using Microsoft.Sbom.Common.Config;
+using Microsoft.Sbom.Extensions;
+using Microsoft.Sbom.Extensions.Entities;
+using Constants = Microsoft.Sbom.Api.Utils.Constants;
+
+namespace Microsoft.Sbom.Api.Manifest.ManifestConfigHandlers;
+
+/// <summary>
+/// Provides the ManifestConfig for the SPDX 3.0 format.
+/// </summary>
+public class SPDX30ManifestConfigHandler : BaseManifestConfigHandler
+{
+    public SPDX30ManifestConfigHandler(
+        IConfiguration configuration,
+        IFileSystemUtils fileSystemUtils,
+        IMetadataBuilderFactory metadataBuilderFactory)
+        : base(configuration, fileSystemUtils, metadataBuilderFactory)
+    {
+    }
+
+    /// <inheritdoc/>
+    protected override ManifestInfo ManifestInfo => Constants.SPDX30ManifestInfo;
+
+    public override bool TryGetManifestConfig(out ISbomConfig sbomConfig)
```

Comment:
The only difference in the implementations of `SPDX22ManifestConfigHandler.TryGetManifestConfig` and `SPDX30ManifestConfigHandler.TryGetManifestConfig` is the `ManifestInfo` you're looking for. You could move the common code into a single method:

```
protected bool TryGetManifestConfig(out ISbomConfig sbomConfig, ManifestInfo ManifestInfo)
```

Then the overridden versions both look the same because you've already overridden `ManifestInfo`:

```
public override bool TryGetManifestConfig(out ISbomConfig sbomConfig)
{
    return base.TryGetManifestConfig(sbomConfig, ManifestInfo;
}
```